bin/package-update-textdomain.js

```diff
@@ -0,0 +1,8 @@
+const wpTextdomain = require( 'wp-textdomain' );
+
+wpTextdomain( 'packages/**/*.php', {
+	domain: 'woocommerce',
+	fix: true,
+	missingDomain: true,
```

Comment:
I wonder if there may be legit cases where a domain is not set in a i18n expression so that WP Core translations are used?

Reply:
Any translatable string will show up for translators on translate.wordpress.org, WordPress can load the default translation, but generally is bad, because the translator editor or the community may not think that the same translation can works in a CMS and Store context.
So we always avoid letting any string without a text domain.